Resources/Prototypes/Entities/Structures/Machines/lathe.yml

```
@@ -19,6 +19,7 @@
           - MachineLayer
   - type: Lathe
   - type: MaterialStorage
+    connectToSilo: true # Goobstation
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)